src/config/rt.rs

```diff
@@ -13,8 +13,10 @@ pub struct RtcBuild {
     pub target: PathBuf,
     /// Build in release mode.
     pub release: bool,
-    /// The output dir for all final assets.
-    pub dist: PathBuf,
+    /// The directory to which plugins should output final assets.
+    pub staging_dist: PathBuf,
+    /// The final resting place where assets will be moved to by Trunk.
+    pub final_dist: PathBuf,
```

Comment:
Totally a nit, and not important, but what do you think about `dist_final` & `dist_stage`???

Reply:
Hmm I am kind of partial to `staging_dist`, although I did think about `dist_staging` too. I prefer `staging` first because it makes you really think about whether you want `staging` or `final` so that it's not just an afterthought.